viewmaster/viewmaster.py

```
@@ -226,7 +269,7 @@ def validate_holdings_paths(abspaths):
             if os.path.exists(parent) and 'holdings' in os.listdir(parent):
                 break
 
-            LOGGER.warn('Holdings not found, pausing', abspath)
+            logger.warn('Holdings not found, pausing', abspath)
```

Ruff (0.14.10)

Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)